static/js/template/services/template.service.js

```diff
@@ -59,9 +58,9 @@
         html = '<md-select class="template-item" ng-model="item.answer" layout="' + item.layout + '">' +
             '<md-option ng-repeat="option in item.values.split(\',\')" value="{{option}}">{{option}}</md-option></md-select>';
       } else if (item.type === 'labeled_input') {
-        html = '<div layout="row" style="word-wrap:break-word"><' + item.sub_type + ' flex="90" layout="column">' + item.values + '</' + 
-                item.sub_type + '><md-input-container flex="10" layout="column">' +
-                '<label>rank</label>' +
+        html = '<div layout="row" style="word-wrap:break-word"><' + item.sub_type + ' flex="75" layout="column">' + item.values + '</' +
+                item.sub_type + '><md-input-container flex="25" layout="column">' +
+                '<label>Enter text here</label>' +
```

Comment:
If we are calling it Labeled Input, then it has to be generic and should allow user to pass any text , so it must have some tangible length.